src/domain/core/src/services/pull_service.rs

```diff
@@ -124,6 +125,8 @@ pub struct PullMultiOptions {
     pub recursive: bool,
     /// Pull all known datasets
     pub all: bool,
+    /// Owner of datasets
+    pub current_account_name: AccountName,
```

Comment:
Adding this parameter is unnecessary.
`PullServiceImpl` already injects `CurrentAccountSubject`, so no need to propagate this value.